core/src/main/java/io/grpc/inprocess/InProcessServerBuilder.java

```diff
@@ -33,11 +33,16 @@
 
 import com.google.common.base.Preconditions;
 
+import com.google.common.util.concurrent.MoreExecutors;
```

Comment:
This import is supposed to go with the previous group of import[s]. That is why the CI was failing.